go/cmd/account.go

```diff
@@ -27,6 +27,7 @@ import (
 // accountCmd represents the account command
 var accountCmd = &cobra.Command{
 	Use: "account",
+    Short: "Create a new account or see account information.",
```

Comment:
The rest is using tab indenting.

Reply:
oh nice spotted! replaced space with tab